website/content/en/preview/contributing/development-guide.md

```diff
@@ -61,6 +61,24 @@ make presubmit # run codegen, lint, and tests
 
 If you are only interested in building the Karpenter images and not deploying the updated release to your cluster immediately with Helm, you can run
 
+*Note: that this will produce a build with the latest version of https://github.com/kubernetes-sigs/karpenter.
+
+```bash
+export $
```

Comment:
Did you mean to leave something here? Or does this do something I'm not aware of

Reply:
Ah yeah, I was debating if I wanted to explicitly add setting `$KO_DOCKER_REPO` but that's already called out earlier. Let me update it and remove